manifest_reprocessing.py

```diff
@@ -0,0 +1,81 @@
+from gladier.defaults import GladierDefaults
+
+
+def ls_dir(data):
+    import os
+    return data['path']
+    # return os.path.listdir('.')
+    # return os.listdir(os.path.dirname(data['path']))
```

Comment:
This function is a stand-in for the 'execution' function, that will need to be replaced with qmap/corr/plotting/pilot. It only serves as a proof of concept for executing after a manifest does a transfer.